core/src/main/java/com/radixdlt/consensus/ConsensusHasher.java

```diff
@@ -113,6 +113,10 @@ public static HashCode toHash(
         } else {
           outputStream.writeByte(0); // 1 byte
         }
+        if (committedHeader.nextProtocolVersion().isPresent()) {
```

Comment:
Can use `ifPresent`?

Reply:
I'd have to handle `write`'s IOException, I think it's better as is